labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,47 @@
+function startGame() {
+  const play = prompt("Would you like to play?");
+  if (play === "yes") {
+    var playerName = prompt("Enter you player name");
+    startCombat(playerName);
+  }
+}
+
+function gameEnd(playerHP, winCount) {
+  if ((winCount === 3) && (playerHP < 0)) {
+    console.log("The game has ended with a draw.");
+  } else if (winCount === 3) {
+    console.log("You have defeated the Almighty Grant!!");
+  } else if (playerHP <= 0) {
+    console.log("YOU LOST!! The Almighty Grant has won, taken over the planet, and destroyed humanity.");
+  }
+}
+
+function getDamage() {
+  const min = 1;
+  const max = 5;
+  return (Math.floor(Math.random() * (max - min + 1) + min));
+}
+
+function startCombat(playerName) {
```

Comment:
This is still a pretty beefy function. You might consider breaking out the "attack" portion.